src/renderer/components/pool/PoolDetails.tsx

```diff
@@ -103,3 +103,5 @@ export const PoolDetails: React.FC<Props> = ({
     </Styled.Container>
   )
 }
+
+export default PoolDetails
```

Comment:
may we rethink using of default exports in the application ? in general people around suppose that this is not a good idea...
https://basarat.gitbook.io/typescript/main-1/defaultisbad

https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad

https://medium.com/@rajeshnaroth/avoid-es6-default-exports-a24142978a7a

Reply:
Valid point :+1:

Reply:
_I'm going to revert these kind of changes with next PR..._

Reply:
^  https://github.com/thorchain/asgardex-electron/pull/536